starter-code/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  render() {
+    return (
+    <div className="App">
+    <div className="top-section">
+      <div className="header">
```

Comment:
rather than giving it a class name you could have just used the ``` <header>``` tag. It helps structure your code and makes it easier to follow by having less class names